### PR TITLE
#20 add privacy provider

### LINKED ISSUE #20: No privacy provider
Failing tests due to no privacy provider

### PR DESCRIPTION
**Pull request by x-iy**
Fixes #20

### CODE REVIEW

File: classes/privacy/provider.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+use core_privacy\local\metadata\null_provider;
+use core_privacy\local\legacy_polyfill;
+/**
+ * Class provider
+ * @author    Ilya Tregubov (ilyatregubov@catalyst-au.net)
+ * @copyright 2018 Catalyst IT
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class provider implements null_provider {
+    use legacy_polyfill;
```
Comment (jwalits) [inline classes/privacy/provider.php]: do you really need this class? and the above used `use core_privacy\local\legacy_polyfill;` line?

File: classes/privacy/provider.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+/**
+ * Privacy provider.
+ *
+ * @package   tool_emailutils
+ * @author    Ilya Tregubov (ilyatregubov@catalyst-au.net)
```
Comment (jwalits) [inline classes/privacy/provider.php]: you might want to update this your name and also update the year for the `@Copyright`

File: classes/privacy/provider.php (hunk 1 above)
Comment (x-iy) [reply classes/privacy/provider.php]: changed as per here: https://moodledev.io/docs/apis/subsystems/privacy#example

File: classes/privacy/provider.php (hunk 2 above)
Comment (x-iy) [reply classes/privacy/provider.php]: updated :)

**Pull request merged**